include/seqan3/utility/views/chunk.hpp

```diff
@@ -250,11 +250,20 @@ class chunk_view<urng_t>::basic_input_iterator :
             return tmp;
         }
 
+// https://github.com/seqan/seqan3/pull/3102
+#if SEQAN3_COMPILER_IS_GCC && (__GNUC__ > 12)
+        //!\brief Compare to the sentinel type (same as sentinel type of the underlying range).
+        friend constexpr bool operator==(input_helper_iterator const & lhs, sentinel_t const &) noexcept
```

Comment:
```suggestion
        friend constexpr bool operator==(input_helper_iterator const & lhs, sentinel_t) noexcept
```
According to specification https://eel.is/c++draft/range.chunk.inner.iter the sentinel object is passed by value. I suggest, we do it like this as well in order to avoid possible confusions later on?